### DISCUSSION TITLE
导入配置

### DISCUSSION THREAD

**Discussion by Dingjinjin-chihiro** [nshen/learn-neovim-lua#120] [Q&A]
请问怎么能j将配置 一键导入到自己的nvim 里来鸭

**Comment by nshen** [comment]
一键导入感觉会有点麻烦，要写个安装脚本，我还不太会，期待大佬分享，我能想到的步骤

1. 检查常见命令是否安装，git, curl, ripgrep....
2. 备份用户现有配置  `~/.config/nvim/`
3. clone 我们的配置
4. install packer.nvim
5. 想办法执行 PackerSync 安装插件
6. 想办法执行Treesitter 安装语法高亮
7. 想办法执行 lsp 安装
